src/agent/src/rpc.rs

```diff
@@ -1641,21 +1641,26 @@ fn setup_bundle(cid: &str, spec: &mut Spec) -> Result<PathBuf> {
     let config_path = bundle_path.join("config.json");
     let rootfs_path = bundle_path.join("rootfs");
 
-    fs::create_dir_all(&rootfs_path)?;
-    BareMount::new(
-        &spec_root.path,
-        rootfs_path.to_str().unwrap(),
-        "bind",
-        MsFlags::MS_BIND,
-        "",
-        &sl!(),
-    )
-    .mount()?;
+    let does_bundle_exist = Path::new(&config_path).is_file();
+    info!(sl!(), "Does the bundle exist {:?}", does_bundle_exist);
+    info!(sl!(), "The config_path is {:?}", config_path);
+
+    if !does_bundle_exist {
```

Comment:
A little of a nit, but I'd rather check for the rootfs existence, because we're creating the rootfs path if the bundle is not there. Something like:

```Rust
if !Path::new(&rootfs_path).exists() {
}
```

Reply:
Updated - I've no problems with nitpicking :)